### PR TITLE
Vet Update

### PR DESCRIPTION
**Pull request by ogiogiovictor**
# Vet clinic database: query and update animals table

## Agregate queries and update for vet-clinics
- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.
alter-table

avg

byspecies

count

DELETE

escape attemp

escape1

rollback

savepoint

species

updated species

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -20,4 +20,57 @@ SELECT * from animals WHERE neutered = true;
...
+
+-- Query Aggrecation
+SELECT COUNT(*) FROM animals;
+
+--What is the average weight of animals?
+SELECT AVG(weight_kg) FROM animals;
+
+-- Who escapes the most, neutered or not neutered animals?
+
+
```
Comment (omar-muhamad) [inline queries.sql]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=How%20many%20animals%20have%20never%20tried%20to%20escape%3F), Please add a query that answers the question `How many animals have never tried to escape?`. 
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Who%20escapes%20the%20most%2C%20neutered%20or%20not%20neutered%20animals%3F), Please add a query that answers the question `Who escapes the most, neutered or not neutered animals?`.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @ogiogiovictor,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (ogiogiovictor) [reply queries.sql]: # Added both queries
image

**Review by ShadyShawkat — APPROVED** [review]
## Hi @ogiogiovictor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**